test/api/options-run-scripts.js

```diff
@@ -95,6 +95,25 @@ describe("API: runScripts constructor option", () => {
     });
   });
 
+  describe("<noscript> childs", () => {
+    it("should be considered as Text when runScripts is set to \"dangerously\"", () => {
+      const { document } = new JSDOM(
+        `<body><noscript><div></div></noscript></body>`,
+        { runScripts: "dangerously" }
+      ).window;
+
+      assert.strictEqual(document.querySelector("noscript").children.length, 0);
+      assert.strictEqual(document.querySelector("noscript").innerHTML, "<div></div>");
+    });
+    it("should be reachable when runScripts is not set to \"dangerously\"", () => {
+      const dom = new JSDOM(`<body><noscript><div></div></noscript></body>`).window;
+      const { document } = dom.window;
+
+      assert.strictEqual(document.querySelector("noscript").children.length, 1);
+      assert.instanceOf(document.querySelector("noscript").children[0], dom.window.HTMLDivElement);
+    });
```

Comment:
Could you add a test for `outside-only` also? Should be the same as "not set to `dangerously`".

Reply:
I added and pushed :)